modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -222,7 +228,7 @@ export default {
       return tabs;
     },
     modalTitle() {
-      return `Edit ${this.moduleOptions.label || ''}`;
+      return this.copyOf ? `Copy ${this.moduleOptions.label || ''}` : `Edit ${this.moduleOptions.label || ''}`;
```

Comment:
This should be New _PIECENAME_ after copying (and also when its actually New, which still says Edit)